app/frontend/src/views/admin/Submission.vue

```diff
@@ -1,570 +1,37 @@
-                      <v-icon color="red" v-else>close</v-icon>
-                    </td>
-                  </tr>
-                </tbody>
-              </template>
-            </v-simple-table>
-
-            <h4>Housekeeping for Ill-Workers</h4>
-            <v-simple-table dense>
-              <template>
-                <tbody>
-                  <tr>
-                    <td>Is able to perform adequate housekeeping for a self isolated worker</td>
-                    <td>
-                      <v-icon color="green" v-if="ipcPlanData.ipcPlan.infectedHousekeeping">check</v-icon>
-                      <v-icon color="red" v-else>close</v-icon>
-                    </td>
-                  </tr>
-                </tbody>
-              </template>
-            </v-simple-table>
-
-            <h4>Waste Management for Ill-Workers</h4>
-            <v-simple-table dense>
-              <template>
-                <tbody>
-                  <tr>
-                    <td>Is able to perform waste management for supporting a self-isolated worker</td>
-                    <td>
-                      <v-icon color="green" v-if="ipcPlanData.ipcPlan.infectedWaste">check</v-icon>
-                      <v-icon color="red" v-else>close</v-icon>
-                    </td>
-                  </tr>
-                </tbody>
-              </template>
-            </v-simple-table>
-
-            <h2 class="mt-8">Certification</h2>
-            <v-simple-table dense>
-              <template>
-                <tbody>
-                  <tr>
-                    <td>Certified this information to be accurate</td>
-                    <td>
-                      <v-icon
-                        @click="showFv = true;"
-                        color="green"
-                        v-if="ipcPlanData.ipcPlan.certifyAccurateInformation"
-                      >check</v-icon>
-                      <v-icon color="red" v-else>close</v-icon>
-                    </td>
-                  </tr>
-
-                  <tr>
-                    <td>Agreed that farm will be subject to a site inspection</td>
-                    <td>
-                      <v-icon color="green" v-if="ipcPlanData.ipcPlan.agreeToInspection">check</v-icon>
-                      <v-icon color="red" v-else>close</v-icon>
-                    </td>
-                  </tr>
-                </tbody>
-              </template>
-            </v-simple-table>
-            <div v-if="showFv">
-              <br />
-              <br />
-              Form Version: {{ ipcPlanData.ipcPlan.formVersion }}
-            </div>
-          </v-container>
-        </v-card>
+        <h1>{{ business.name }}</h1>
+        <h3>Submitted: {{ new Date(ipcPlan.createdAt).toLocaleString() }}</h3>
+
+        <v-row>
+          <v-col cols="12" md="8" class="pl-0 pt-0">
+            <AdminReviewSubmission />
+          </v-col>
+          <v-col cols="12" md="4">
```

Comment:
i guess you can put in ` order="1"` to have the status stuff show up above on mobile screens

Reply:
Yes great point, would want the submission grid to pull above on mobile.